desc/protoparse/parser.go

```diff
@@ -868,7 +868,11 @@ func (r *parseResult) asMethodDescriptor(node *methodNode) *dpb.MethodDescriptor
 	if node.output.streamKeyword != nil {
 		md.ServerStreaming = proto.Bool(true)
 	}
-	if len(node.options) > 0 {
+	// protoc always adds a MethodOptions if there are brackets
```

Comment:
I don't think this latest formulation really warrants a comment to explain it.

Reply:
I think it does, since we do `len(node.options) > 0` in all other cases